packages/core/src/workflow/create-tx/CreateBitcoinTx.ts

```diff
@@ -162,15 +165,14 @@ export class CreateBitcoinTx {
   create(): UnsignedBitcoinTx {
     return {
       fee: this.fees.number.toNumber(),
-      inputs: this.tx.from.map((item) => {
-        return {
-          address: item.address,
-          amount: this.amountDecoder(item.value).number.toNumber(),
-          entryId: this.source.id,
-          txid: item.txid,
-          vout: item.vout,
-        };
-      }),
+      inputs: this.tx.from.map(({ address, sequence, txid, value, vout }) => ({
+        address,
+        txid,
+        vout,
+        amount: this.amountDecoder(value).number.toNumber(),
+        entryId: this.source.id,
+        sequence: sequence == null ? MAX_SEQUENCE - 15 : sequence < MAX_SEQUENCE ? sequence + 1 : sequence,
```

Comment:
It's too hard to read. Please replace with a function. 

Also `MAX_SEQUENCE - 15` looks too arbitrary, it's better to have `MAX_SEQUENCE - 0x0f` here, or even make a constant `const MIN_SEQUENCE = MAX_SEQUENCE - 0x0f` and use it here.

Reply:
and you probably wanted `sequence < MAX_SEQUENCE ? sequence + 1 : MAX_SEQUENCE`